scripts/ci/jenkins_pipeline_csi

```diff
@@ -25,9 +33,47 @@ pipeline {
                 sh './scripts/run_yamlcheck.sh'
             }
         }
-        stage ('CSI-controller & node: Build and push images') {
+        stage ('Build docker images') {
+            steps {
+                script {
+                    sh 'bash ./scripts/ci/run_variables_verification.sh'
+                    branch = sh(script:"echo $GIT_BRANCH | sed 's|/|.|g'",returnStdout:true).trim()
+                    specific_tag = "${IMAGE_VERSION}_b${BUILD_NUMBER}_${branch}"
+                    if (latest_tag.contains(GIT_BRANCH)) {
+                        controller_image_latest = docker.build("${CSI_CONTROLLER_IMAGE}", "--build-arg VERSION=${IMAGE_VERSION} --build-arg BUILD_NUMBER=${BUILD_NUMBER} -f ${Dockerfile_controller} .")
```

Comment:
do we have to have separate build commands for latest tags? I don't think the original code had

Reply:
That how Jenkins works, you can't build the same image in the command with two tagges, I can add a docker tag instead of two docker build for each image. And I think that we should keep the variable even if we will have one build command, because it will be easier to change it if the Dockerfile path will change.

Reply:
what do you mean "how Jenkins works"?
can't you just build once (i.e. `controller_image = docker.build(...)`) and then push two tags (i.e. `controller_image.push("${controller_registry}:$specific_tag")` and `controller_image.push("${controller_registry}:latest`)?

Reply:
because this is not the syntax this plugin works, I checked in a lot of places and everyone said that in order to build one image with two tags in Jenkins, the only way is how I did it. 
You can't do:
`controller_image.push("${controller_registry}:$specific_tag")
controller_image.push("${controller_registry}:latest")`

I can try and check it again

Reply:
do check again, and please add your links here.
this is what I found:
https://stackoverflow.com/a/53858257

Reply:
done